core/src/components/textarea/test/highlight/textarea.e2e.ts

```diff
@@ -347,3 +347,123 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ directions: ['ltr'], modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('textarea: highlights'), () => {
+    test.describe('textarea: no fill', () => {
```

Comment:
I copied this from input but maybe it doesn't make sense to have both since "no fill" defaults to outline for the ionic theme

Reply:
I agree, I would remove this one